### PR TITLE
Lecture 03 (abf149 / Andrew Feldman)

### PR DESCRIPTION
**Pull request by abf149**

### CODE REVIEW

File: notes/lecture-03-abf149.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+    - The increasing cost of memory combined with increasing model size means that storage cost is a key reason for minimizing model size
+    - It is also more costly to build memories which support higher bandwidth, and as models expand to have more weights, the bandwidth required to move move chunks of weight within the processor memory hierarchy (main memory, cache, register file) increases. This too motivates minimizing model size
+    - Downloading a DNN onto a mobile device is another point where model size is impactful; large model size can mean the difference between being able to download a DNN model quickly over mobile data vs taking a long time to transfer the model and/or requiring WiFi
+    - Example - 10 million weights may be two large to store on a phone or conveniently transfer to a phone, especially over Cellular Data
+- Model execution time
+    - The scaling behavior of execution time with respect to model size is a function of the arithmetic operations comprising the neural network
+- model inference energy
```
Comment (zhijian-liu) [inline notes/lecture-03-abf149.md:43]: "model" => "Model"

File: notes/lecture-03-abf149.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+### Introducing pruning through a neurobiological analogy
+
+In neurobiology axonal connections are understood to play a role in representing and computing on information in the brain. The number of axonal connections ramps up after birth, stays high for a period of time, until eventually a very large number of connections are destroyed toward the end of adolescence and into adulthood. Scientists assume that the brain does this in order to facilitate aggressive learning early in life, followed by honing in on only the most critical knowledge later in life. This latter stage of destroying axonal connections is referred to in neurobiology as ``pruning'' of the neuronal axons.
```
Comment (zhijian-liu) [inline notes/lecture-03-abf149.md:67]: ``'' is used in Tex. Please use the normal quotes ("") in Markdown. Make sure to fix all occurrences.

**Review by zhijian-liu — APPROVED** [review]
Thanks for your contribution! The scribe looks great. Please fix the format issues and feel free to merge this PR.

File: notes/lecture-03-abf149.md (hunk 1 above)
Comment (abf149) [reply notes/lecture-03-abf149.md]: Done

File: notes/lecture-03-abf149.md (hunk 2 above)
Comment (abf149) [reply notes/lecture-03-abf149.md]: Done

**Comment by abf149** [comment]
Thanks, I have made the requested changes.

**Comment by zhijian-liu** [comment]
Thank you!

**Pull request merged**